app/client/views/login/loginCtrl.js

```diff
@@ -46,13 +46,23 @@ angular.module('reg')
 
       $scope.sendResetEmail = function() {
         var email = $scope.email;
-        AuthService.sendResetEmail(email);
-        sweetAlert({
+        var dialogue = {
```

Comment:
can we perform the regex check beforehand instead of overriding the `dialogue` var if it's not a valid email?

i.e.
```
if (re.test(email)) {
  sendResetEmail();
  sweetAlert({ title, text ... });
} else {
  sweetAlert({ title, text ... });
}
```